builtin/logical/ssh/path_config_ca.go

```diff
@@ -112,39 +164,37 @@ func (b *backend) pathConfigCAUpdate(req *logical.Request, data *framework.Field
 		return nil, fmt.Errorf("failed to generate or parse the keys")
 	}
 
-	publicKeyEntry, err := req.Storage.Get("config/ca_public_key")
+	storedPublicKey, err := caKey(req.Storage, "public")
 	if err != nil {
-		return nil, fmt.Errorf("failed while reading ca_public_key: %v", err)
+		return nil, fmt.Errorf("failed to read CA public key: %v", err)
 	}
 
-	privateKeyEntry, err := req.Storage.Get("config/ca_bundle")
+	storedPrivateKey, err := caKey(req.Storage, "private")
 	if err != nil {
-		return nil, fmt.Errorf("failed while reading ca_bundle: %v", err)
+		return nil, fmt.Errorf("failed to read CA private key: %v", err)
 	}
 
-	if publicKeyEntry != nil || privateKeyEntry != nil {
+	if storedPublicKey != "" || storedPrivateKey != "" {
```

Comment:
Is it possible to get into a state where only one of these is set? If so can we protect against that happening?